backend/main.go

```diff
@@ -4,34 +4,89 @@ import (
 	"fmt"
 	"log"
 	"net/http"
-
+	"io/ioutil"
 	"github.com/joho/godotenv"
 	db "github.com/ubclaunchpad/news-sentiment/db"
+	"github.com/gorilla/mux"
 )
 
-func hello(w http.ResponseWriter, req *http.Request) {
 
-	fmt.Fprintf(w, "Hello world!\n")
+
+func homePage(w http.ResponseWriter, r *http.Request){
+    fmt.Fprintf(w, "HOME: News Analysis App")
+    fmt.Println("Endpoint Hit: homePage")
 }
 
-func headers(w http.ResponseWriter, req *http.Request) {
 
-	for name, headers := range req.Header {
-		for _, h := range headers {
-			fmt.Fprintf(w, "%v: %v\n", name, h)
-		}
-	}
+
+
+
+//GET: endpoint for a single user
+func getUser(w http.ResponseWriter, req *http.Request){
+
+	vars := mux.Vars(req)
+	id := vars["id"]
+	fmt.Fprintf(w, "Requesting details for user: " + id)
+
+}
+
+
+//POST: endpoint to add a single user
+func addUser(w http.ResponseWriter, req *http.Request){
+	
+    reqBody, _ := ioutil.ReadAll(req.Body)
+    fmt.Fprintf(w, "%+v", string(reqBody))
+
 }
 
+
+//GET: endpoint for a single article
+func getArticle(w http.ResponseWriter, req *http.Request){
+
+	vars := mux.Vars(req)
+	id := vars["id"]
+	fmt.Fprintf(w, "Requesting details for article: " + id)
+
+}
+
+
+//POST: endpoint to add a single article
+func addArticle(w http.ResponseWriter, req *http.Request){
+
+    reqBody, _ := ioutil.ReadAll(req.Body)
+    fmt.Fprintf(w, "%+v", string(reqBody))
+
+}
+
+
+
+
+//Driver for all routes
+func handleRequests(){
+
+	router := mux.NewRouter().StrictSlash(true)
```

Comment:
I love that we're using `mux`! It's amazing!

Reply:
It's really clean!